operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -208,6 +209,15 @@
   @Range(minimum = 0)
   private Integer maxClusterConcurrentStartup;
 
+  @Description(
+          "The default maximum WebLogic Server instances that a cluster will shutdown in parallel when it is being "
```

Comment:
Surprisingly, this text is slightly different. "The default maximum number of ". "shutdown" -> "shut down". "attribute" -> "field". "Defaults to 1." (add period).